specifyweb/frontend/js_src/lib/components/Atoms/Icons.tsx

```diff
@@ -77,6 +77,7 @@ export const icons = {
   hashtag: <svg aria-hidden className={iconClassName} fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path clipRule="evenodd" d="M9.243 3.03a1 1 0 01.727 1.213L9.53 6h2.94l.56-2.243a1 1 0 111.94.486L14.53 6H17a1 1 0 110 2h-2.97l-1 4H15a1 1 0 110 2h-2.47l-.56 2.242a1 1 0 11-1.94-.485L10.47 14H7.53l-.56 2.242a1 1 0 11-1.94-.485L5.47 14H3a1 1 0 110-2h2.97l1-4H5a1 1 0 110-2h2.47l.56-2.243a1 1 0 011.213-.727zM9.03 8l-1 4h2.938l1-4H9.031z" fillRule="evenodd" /></svg>,
+  history: <svg aria-hidden className={iconClassName} fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><g><g><path clipRule="evenodd" d="M9.9,17.4c-2.1,0-3.8-0.7-5.2-2.2c-1.3-1.3-2-2.8-2.1-4.6c0-0.4,0.2-0.7,0.6-0.7l0,0c0.3,0,0.6,0.2,0.6,0.6 c0.1,1.5,0.7,2.8,1.7,3.8c1.2,1.2,2.6,1.8,4.4,1.8s3.2-0.6,4.4-1.9s1.8-2.7,1.8-4.4c0-1.7-0.6-3.1-1.9-4.3s-2.7-1.8-4.4-1.8 C9,3.8,8.1,4,7.3,4.5c-0.8,0.5-1.5,1-2.1,1.7h1.6c0.3,0,0.6,0.2,0.6,0.6l0,0c0,0.4-0.3,0.7-0.7,0.7H3V3.7c0-0.4,0.3-0.6,0.6-0.6 l0,0c0.3,0,0.6,0.3,0.6,0.6v1.6c0.7-0.8,1.6-1.5,2.6-2s2-0.7,3.1-0.7c1,0,2,0.2,2.9,0.6c0.9,0.4,1.7,0.9,2.4,1.6 c0.7,0.7,1.2,1.4,1.6,2.3s0.6,1.9,0.6,2.9s-0.2,2-0.6,2.9c-0.4,0.9-0.9,1.7-1.6,2.4c-0.7,0.7-1.5,1.2-2.4,1.6 C11.9,17.2,11,17.4,9.9,17.4z M12.1,12.9l-2.3-2.3C9.5,10.4,9.4,10,9.4,9.7V6.4c0-0.3,0.2-0.5,0.5-0.5H10c0.3,0,0.6,0.2,0.6,0.6 v3.3l2.4,2.3c0.2,0.2,0.2,0.7,0,0.9l0,0C12.7,13.2,12.4,13.2,12.1,12.9z" fillRule="evenodd"/></g><g><path clipRule="evenodd" d="M9.9,17.4c-2.1,0-3.8-0.7-5.2-2.2c-1.3-1.3-2-2.8-2.1-4.6c0-0.4,0.2-0.7,0.6-0.7l0,0c0.3,0,0.6,0.2,0.6,0.6 c0.1,1.5,0.7,2.8,1.7,3.8c1.2,1.2,2.6,1.8,4.4,1.8s3.2-0.6,4.4-1.9s1.8-2.7,1.8-4.4c0-1.7-0.6-3.1-1.9-4.3s-2.7-1.8-4.4-1.8 C9,3.8,8.1,4,7.3,4.5c-0.8,0.5-1.5,1-2.1,1.7h1.6c0.3,0,0.6,0.2,0.6,0.6l0,0c0,0.4-0.3,0.7-0.7,0.7H3V3.7c0-0.4,0.3-0.6,0.6-0.6 l0,0c0.3,0,0.6,0.3,0.6,0.6v1.6c0.7-0.8,1.6-1.5,2.6-2s2-0.7,3.1-0.7c1,0,2,0.2,2.9,0.6c0.9,0.4,1.7,0.9,2.4,1.6 c0.7,0.7,1.2,1.4,1.6,2.3s0.6,1.9,0.6,2.9s-0.2,2-0.6,2.9c-0.4,0.9-0.9,1.7-1.6,2.4c-0.7,0.7-1.5,1.2-2.4,1.6 C11.9,17.2,11,17.4,9.9,17.4z M12.1,12.9l-2.3-2.3C9.5,10.4,9.4,10,9.4,9.7V6.4c0-0.3,0.2-0.5,0.5-0.5H10c0.3,0,0.6,0.2,0.6,0.6 v3.3l2.4,2.3c0.2,0.2,0.2,0.7,0,0.9l0,0C12.7,13.2,12.4,13.2,12.1,12.9z" fillRule="evenodd"/></g></g></svg>,
```

Comment:
could you add a short comment before the icon explaining that it's not from heroicons. This is needed because almost every other icon in this file is from heroicons

i.e, something like this would do:
```
// This icon is not from Heroicons. It was drawn by @grantfitzsimmons
```